main.tf

```diff
@@ -114,3 +115,38 @@ resource "null_resource" "consul" {
     command = "bash -e ${path.module}/createDns.sh '${var.product}-${var.env}-${var.staging_slot_name}' 'core-infra-${var.env}' '${path.module}' '${var.ilbIp}' '${var.subscription}'"
   }
 }
+
+# No way we can apply the required properties as part of the ARM template so have to use Azure CLI
+resource "null_resource" "app_service_security" {
+  count    = "${local.hmctsdemo}"
+  triggers {
+    trigger = "${data.azurerm_key_vault_secret.security_aad_tenantId.value}",
+    trigger = "${data.azurerm_key_vault_secret.security_aad_clientId.value}",
+    trigger = "${data.azurerm_key_vault_secret.security_aad_clientSecret.value}"
+  }
+  
+  # configure App Service security
+  provisioner "local-exec" {
+    command = "bash -e ${path.module}/configureSecurity.sh '${var.subscription}' '${var.product}-${var.env}' '${azurerm_resource_group.rg.name}' '${data.azurerm_key_vault_secret.security_aad_tenantId.value}' '${data.azurerm_key_vault_secret.security_aad_clientId.value}' '${data.azurerm_key_vault_secret.security_aad_clientSecret.value}'"
```

Comment:
what happens when this is run on sandbox / not hmctsdemo branch? 

i.e. I assume the key vault secrets aren't resolved and it errors? unless there's something I'm missing